src/Vizkit3DPlugin.hpp

```diff
@@ -109,12 +109,14 @@ class Vizkit3DWidget;
  */
 class VizPluginBase : public QObject
 {
+    friend class PickHandler;
+
     Q_OBJECT
     Q_PROPERTY(QString vizkit3d_plugin_name READ getPluginName)
     Q_PROPERTY(bool enabled READ isPluginEnabled WRITE setPluginEnabled)
     Q_PROPERTY(bool KeepOldData READ isKeepOldDataEnabled WRITE setKeepOldData)
     Q_PROPERTY(int MaxOldData READ getMaxOldData WRITE setMaxOldData)
-    Q_PROPERTY(QStringList frame READ getVisualizationFrames WRITE setVisualizationFrame)
+    Q_PROPERTY(QStringList frame READ getVisualizationFrames WRITE setVisualizationFrames)
```

Comment:
I agree, do you have any other name you would prefer? We cannot use the old name because qtruby has a bug and accidentally calls the overloaded method with a string as argument resulting in a segfault.